baselines/FedPer/FedPer/run_figures.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+### UNCOMMENT THE FOLLOWING LINES TO RUN THE EXPERIMENTS ###
+
+# CIFAR10 Mobile
+#python -m FedPer.main --config-path conf/cifar10/mobile --config-name fedper_10 &&
+#python -m FedPer.main --config-path conf/cifar10/mobile --config-name fedper_8 &&
+#python -m FedPer.main --config-path conf/cifar10/mobile --config-name fedper_4 &&
```

Comment:
these configs only differ by one parameter. Wouldn't it be better to have a single config (e.g. `fedper.yaml`) and override that parameter? for example this would look like:
```
python -m FedPer.main --config-path conf/cifar10/mobile --config-name fedper dataset.num_classes=4 # or 8 or 10
```